gnovm/pkg/gnolang/coverage.go

```diff
@@ -0,0 +1,642 @@
+package gnolang
```

Comment:
I don't think the implementation of `CoverageData`, and others needs to be in the `pkg/gnolang` package. Maybe we can move it into a sub-package, or even outside the `gnolang` directory?

cc @thehowl

Reply:
The more I read this code, the more I'm convinced it shouldn't be in `pkg/gnolang` at all

Reply:
It seems moving to position it at the same level as the repl or gnofmt directories more appropriate.

I’ll work on reducing the dependency on the Machine and then move forward to relocate the package.